src/bot.jl

```diff
@@ -1,55 +1,66 @@
+  Defaults to `false`. Use this feature if SnoopCompile benchmark or your CI fails.
+  This feature also prints the errors, which you can fix by `blacklist`ing the offending statements
+  and then disabling `exhaustive` for future runs.
 
-- `os`: A vector of of Strings (or RegExp) to give the list of os that you want to generate precompile signatures for. Each element will call a `Sys.is\$eachos()` function.
+- `os`: A vector of of Strings (or RegExp) to support with precompile statements.
+  Each element will call a `Sys.is\$eachos()` function.
 
 Example: `os = ["windows", "linux"]`
 
-- `else_os`: If you want to use a specific os for any other os, give `else_os` the name of that os. If you don't want to precompile for other os pass nothing.
+- `else_os`: If you want to use a specific operating system's precompile file as the default,
+  set `else_os` to the name of that os.
+  Setting this to `nothing` disables precompilation on any operating system other than those
+  explicitly listed in `os`.
 
 Example: `else_os = "linux"`
 
-- `version`: A vector of of versions to give the list of versions that you want to generate precompile signatures for.
+- `version`: A vector of of Julia versions used to generate precompile signatures.
 
 Example: `version = [v"1.1", v"1.4.1"]`
 
-It is assumed that the generated precompile signatures are valid for patch versions of Julia (e.g. giving v"1.4.2" assumses v"1.4.0" to v"1.4.9").
+It is assumed that the generated precompile signatures are valid for patch versions of Julia (e.g. giving v"1.4.2" supports v"1.4.0" and v"1.4.9").
 
-- `else_vresion`: If you want to use a specific version for any other version, give `else_version` the name of that version.
+- `else_version`: the Julia version used to generate the fallback (default) signatures.
 
 Example: `else_version = v"1.4.1"`
 
-- `package_path`: path to the main `.jl` file of the package (similar to `pathof`). Default path is `pathof_noload(package_name)`.
+- `package_path`: path to the main `.jl` file of the package (similar to `pathof`).
+  Default path is `pathof_noload(package_name)`.
 
-- `precompiles_rootpath`: the path where precompile files are stored. Default path is "\$(dirname(dirname(package_path)))/deps/SnoopCompile/precompile".
+- `precompiles_rootpath`: the path where precompile files are stored.
+  Default path is "\$(dirname(dirname(package_path)))/deps/SnoopCompile/precompile".
 
-- `subst` : A vector of pairs of Strings (or RegExp) to replace a packages precompile setences with another's package like `["ImageTest" => "Images"]`.
+- `subst` : A vector of pairs of Strings (or RegExp) to replace a packages precompile statements
+  with another's package like `["ImageTest" => "Images"]`.
 
-- `tmin`: Methods that take less time than `tmin` will not be reported. Defaults to 0.0.
+- `tmin`: Methods that take less time than `tmin` to be inferred will not be added to the
+  precompile statements. Defaults to 0.
```

Comment:
Do you think we should use something bigger as `tmin`? The default value in `@snoopi` is `0`, and if we want to set something bigger in `BotConfig`, we should be consistent with `@snoopi`.

Reply:
Maybe so. I worry that anything other than 0 is essentially arbitrary, though. For reference, I essentially always use `tmin=0.01`.

Reply:
`t=0.0` has its own uses. Especially, when we develop the PackageCompilerBot, we can use all those generated statements for compiling to machine code.

But currently, in this setup, probably we should use something bigger. `tmin=0.01` sounds good.